### DISCUSSION TITLE
Mention custom date & time format in the FAQ

### DISCUSSION THREAD

**Discussion by JohnnyCalavera** [SpiralLinux/SpiralLinux-project#29] [General]
Hi Sam,

thank you for providing nice looking and easy to use desktop debian iso images. We support public schools in Germany and encountered a tiny issue the moment we tried to change the clock settings in cinnamon (the toggles for showing seconds and the one for 24h instead of a.m./p.m. did not have any effect). We tried XFCE next with a similar behaviour, but in the XFCE settings it became obvious that a custom date and time format overrides every tweak we tried. 
So maybe you could add a line of information to the FAQ since more than a few people use 24h clocks and most of them probably won't check the config scripts ^^ 

Kind regards

**Comment by geckolinux** [comment]
Hi Johnny, thanks a lot for your kind and useful feedback. So do I understand correctly the the school you support is using or will be using SpiralLinux?

Sorry for the inconvenience of the the custom date/time format. Is the 24h time vs AM/PM issue the only one, or are you also seeing the wrong date format? (i.e. `28 Jun 2022` vs `June 28, 2022` ? The date format should be correct according to the language/locale that was selected during installation. But you are correct that in some editions the AM/PM override is included. To enable 24h time in Cinnamon:

![Screenshot from 2022-06-28 13-04-07](https://user-images.githubusercontent.com/2692138/176253059-afa4915d-0965-4a6b-b582-3419596e6d6d.png)

And in the XFCE flavour:

![Screenshot from 2022-06-28 13-12-08](https://user-images.githubusercontent.com/2692138/176253350-38a0e8ee-fbdc-4082-a35f-0ecd89516a5d.png)

Is that what you were looking for? Thanks again.

**Reply by JohnnyCalavera** [reply]
Thanks for the reply. I should have mentioned that we were able to solve this, after we noticed the custom format. What I was trying to say is that the we only looked at the "date & time" settings page on the right of your screenshot, but the toggles didn't have any effect. We did not check the calendar settings where we could have disabled the custom format which is set by default in SpiralLinux. 
So our intention was to point out: maybe you could mention in the FAQ that there's a custom time format preconfigured just so some users can easily find this information ^^ 
Thanks again - we are really happy to use a distro with official debian repos now and your btrfs configuration makes our lives easier as well. (We used Linux Mint and Zorin OS Education in the past, but being based on Ubuntu we're afraid of what Canonical might do next with their distribution - Mint even started a partnership with Mozilla to provide Firefox as non-Snap / deb package in the futute for example.)

**Reply by geckolinux** [reply]
Hi Johnny, thanks for the additional information. OK, that makes sense to add that to the FAQ, thanks for the suggestion.

I totally understand your concern about the upstream distro making radical changes. That's what I also really appreciate about Debian, they have really earned their reputation for being consistent and predictable. I hope that together with the SpiralLinux configuration and delivery method it will be a stable and reliable solution for your needs.
